client/src/components/pages/game.tsx

```diff
@@ -46,23 +48,50 @@ export function GamePage() {
     return Socials.merge(edition?.socials, game?.socials);
   }, [edition, game]);
 
+  const isDashboard = !(edition && game);
+
   return (
     <>
-      <div className="lg:h-[88px] w-full flex flex-col p-4 gap-4 lg:p-6 lg:pb-0 border-b border-background-200 lg:border-none">
+      <div
+        className={cn(
+          "lg:h-[88px] w-full flex flex-col gap-4 lg:p-6 lg:pb-0 border-b border-background-200 lg:border-none",
+          isDashboard ? "p-0" : "p-4",
+        )}
+      >
         <div className="flex items-start justify-between">
-          <div className="flex gap-3 items-center overflow-hidden">
-            <Thumbnail
-              icon={edition?.properties.icon || game?.properties.icon || arcade}
-              size="xl"
-              className="min-w-16 min-h-16"
-            />
-            <div className="flex flex-col gap-2 overflow-hidden">
-              <p className="font-semibold text-xl/[24px] text-foreground-100 truncate">
-                {game?.name ?? "Arcade Dashboard"}
-              </p>
-              <Editions />
-            </div>
-          </div>
+          {
+            isMobile && !isDashboard ? ( // Mobile view + not dashboard
+              <div className="flex gap-4 items-center overflow-hidden">
+                <Thumbnail
+                  icon={edition?.properties.icon || game?.properties.icon}
+                  size="xl"
+                  className="min-w-16 min-h-16"
+                />
+                <div className="flex flex-col gap-2 overflow-hidden">
+                  <p className="font-semibold text-xl/[24px] text-foreground-100 truncate">
+                    {game?.name || "Dashboard"}
+                  </p>
+                  <Editions />
+                </div>
+              </div>
+            ) : !isMobile ? ( // Desktop view
```

Comment:
I dont understand while we duplicate the whole block between (mobile + dashboard) and desktop here, the content looks very similar
I'd suggest to just hide the parent div in case `isDashboard && isMobile && "hidden"`